lib/std/gpu.zig

```diff
@@ -81,21 +80,17 @@ pub fn fragmentDepth(comptime ptr: *addrspace(.output) f32) void {
 /// Forms the main linkage for `input` and `output` address spaces.
 /// `ptr` must be a reference to variable or struct field.
 pub fn location(comptime ptr: anytype, comptime loc: u32) void {
-    const code = comptimePrint("OpDecorate %ptr Location {}", .{loc});
-    asm volatile (code
+    asm volatile ("OpDecorate %ptr Location " ++ &[_]u8{'0' + loc}
         :
         : [ptr] "" (ptr),
     );
 }
 
 /// Forms the main linkage for `input` and `output` address spaces.
 /// `ptr` must be a reference to variable or struct field.
-pub fn binding(comptime ptr: anytype, comptime group: u32, comptime bind: u32) void {
-    const code = comptimePrint(
-        \\OpDecorate %ptr DescriptorSet {}
-        \\OpDecorate %ptr Binding {}
-    , .{ group, bind });
-    asm volatile (code
+pub fn binding(comptime ptr: anytype, comptime descriptor_set: u32, comptime bind: u32) void {
+    asm volatile ("OpDecorate %ptr DescriptorSet " ++ &[_]u8{'0' + descriptor_set} ++
+            "OpDecorate %ptr Binding " ++ &[_]u8{'0' + bind}
         :
         : [ptr] "" (ptr),
     );
```

Comment:
I think that this should work

```suggestion
pub fn binding(comptime ptr: anytype, comptime descriptor_set: u32, comptime bind: u32) void {
    asm volatile (
      \\OpDecorate %ptr DescriptorSet $set
      \\OpDecorate %ptr Binding $bind
        :
        : [ptr] "" (ptr),
          [set] "c" (set),
          [bind] "c" (bind),
    );
```
this $constant syntax was added a while ago I think